src/components/calculator.css

```diff
@@ -0,0 +1,107 @@
+.calculator {
+  display: grid;
+  grid-template-rows: auto auto repeat(5, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+  width: 500px;
+  height: 400px;
+  grid-gap: 5px;
+  padding: 10px;
+  border: 1px solid black;
+}
+
+.calculatorDisplay {
+  grid-row: 1 / span 2;
+  grid-column: 1 / span 4;
+  background-color: grey;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: flex-end;
+  padding: 10px;
+  font-size: 36px;
+}
+
+.calculatorDisplay .previous {
+  font-size: 24px;
+}
+
+.calculatorDisplay .current {
+  font-weight: bold;
+}
+
+.calc {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 5px;
+  grid-row: 3 / span 5;
+  grid-column: 1 / span 4;
+}
+
+.calc button:nth-child(4n) {
+  background-color: orange;
+}
+
+.calc button:nth-child(19) {
+  background-color: orange;
+}
+
+.calc button:nth-child(17) {
+  grid-column: span 2;
+}
+
+/*
+.calculatorDisplay {
+  grid-column: 1 / span 4;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-end;
+  background-color: grey;
+  color: white;
+  padding: 10px;
+}
+
+.previous {
+  font-size: 24px;
+  margin-bottom: 10px;
```

Comment:
- [ ] Kindly remove all unused and commented pieces of code from your codebase. This makes your codebase clean and professional.